modules/ROOT/pages/type-definitions/directives/index.adoc

```diff
@@ -1,11 +1,182 @@
+
+.Database mapping directives
+[cols="2,5"]
+|===
+| Directive | Description
+
+| xref::/type-definitions/directives/database-mapping.adoc#_relationship[`@relationship`]
+| Configure  xref::/type-definitions/types/relationships.adoc[relationships] between object types.
+
+| xref::/type-definitions/directives/database-mapping.adoc#_relationship_properties[`@relationshipProperties`]
+a| Required to help you distinguish between interfaces which are used for relationship properties, and otherwise.
+Can only be used on interfaces, as per its definition:
+[source, graphql, indent=0]
+----
+"""Required to differentiate between interfaces for relationship properties, and otherwise."""
+directive @relationshipProperties on OBJECT
+----
+
+| xref::/type-definitions/directives/database-mapping.adoc#type-definitions-node[`@node`]
+| Specifies the configuration of a GraphQL object type which represents a Neo4j node.
+
+| xref::/type-definitions/directives/database-mapping.adoc#type-definitions-alias[`@alias`]
+| Maps a GraphQL schema field to a Neo4j property on a node or relationship.
+
+| xref::/type-definitions/directives/database-mapping.adoc#_declarerelationship[`@declareRelationship`]
+| Configure xref::/type-definitions/types/relationships.adoc[relationships] to be implemented on object types.
+
+|===
+
+== Security
+
+.Security directives
+[cols="2,5"]
+|===
+| Directive | Description
+
+| xref::/type-definitions/directives/security/authentication.adoc[`@authentication`]
+| Requires authentication checks when accessing the type.
+
+| xref::/type-definitions/directives/security/authorization.adoc[`@authorization`]
+| Specifies authorization rules for queries and mutations on the type.
+
+| xref::/type-definitions/directives/security/configuration.adoc#authentication-and-authorization-jwt[`@jwt`]
+| Configure the JWT authentication filter to include an additional JWT claim.
+
+| xref::/type-definitions/directives/security/configuration.adoc#_nested_claims[`@jwtClaim`]
+| Use in combination with `@jwt`. Configure the JWT authentication filter to include an additional nested JWT claim.
+
+| `@subscriptionsAuthorization`
+| Specifies authorization rules for subscriptions on the type.
+
+|===
+
+== Autogeneration
+
+.Autogeneration directives
+[cols="2,5"]
+|===
+| Directive | Description
+
+| xref::/type-definitions/directives/autogeneration.adoc#type-definitions-autogeneration-id[`@id`]
+| Marks a field as the unique ID for an object type, and allows for autogeneration of IDs.
+
+| xref::/type-definitions/directives/autogeneration.adoc#type-definitions-autogeneration-timestamp[`@timestamp`]
+| Flags fields to be used to store timestamps on create/update events.
+
+|===
+
+== Schema configuration
+
+.Schema configuration directives
+[cols="2,5"]
+|===
+| Directive | Description
+
+| xref:/type-definitions/directives/schema-configuration/type-configuration.adoc#_query[`@query`]
+| Limits the availability of Query operations in the library.
```

Comment:
This was something that I noted before in these docs, there is no rule about what terms should be capitalized or not. In other pages, I have the feeling that these same terms are not capitalized when used in a similar way.